src/Popover/__stories__/Popover.stories.js

```diff
@@ -18,13 +18,79 @@ const bodyContent = (
     </Menu>
 );
 
+const someText = <p style={{ padding: '0.5em' }}>This is a simple popover.</p>;
+
 storiesOf('Components|Popover', module)
     .addDecorator(withKnobs)
     .add('Default', () => (
         <Popover
             body={bodyContent}
             control={<Button glyph='navigation-up-arrow' option='light' />} />
     ))
+    .add('Placement', () => (
+        <>
+            <Popover
+                body={someText}
+                control={<Button glyph='navigation-up-arrow' option='light' >Pop to Top-Start</Button>}
+                placement='top-start'
+                style={{ position: 'fixed', top: '10em', left: '35em' }} />
```

Comment:
The placement `style` for each of these is strange.  When I have the inspector open, I am not seeing the bottom row and no scrollbar appears.  How about building a table that is centered?

| Layout | | | | |
|--|--|--|--|--|
|  |  Top Start | Top | Top End |  |
| Left Start | | | | Right Start |
| Left | | | | Right |
| Left End | | | | Right End |
| | Bottom Start | Bottom | Bottom End | |